packages/modular-scripts/package.json

```diff
@@ -6,14 +6,17 @@
     "modular": "build/cli.js"
   },
   "main": "./build/index.js",
+  "types": "dist-types/index.d.ts",
   "engines": {
     "node": ">=12"
   },
   "scripts": {
     "modular": "ts-node src/cli.ts",
-    "clean": "rimraf build",
+    "clean": "rimraf dist-cjs dist-types",
     "prebuild": "yarn clean",
-    "build": "babel --root-mode upward src --out-dir build --extensions .ts --ignore 'src/**/*.test.ts'"
+    "build": "yarn build:js && yarn build:ts",
+    "build:js": "babel --root-mode upward src --out-dir dist-cjs --extensions .ts --ignore 'src/**/*.test.ts'",
+    "build:ts": "tsc --project tsconfig.build.json"
```

Comment:
We can't use modular to build itself?

Reply:
There's an open issue to do this - we don't really support ```bin``` and ```main``` side by side at the moment so it's likely we need some changes to how we build packages.